boa_engine/src/vm/opcode.rs

```diff
@@ -1066,6 +1066,137 @@ impl Opcode {
             Opcode::Nop => "Nop",
         }
     }
+
+    pub fn as_instruction_str(self) -> &'static str {
```

Comment:
I would add a bit of documentation to this function.

Reply:
Ack. I'll add documentation to this function.